tests/iface_loopback_action/iface_loopback_action_helper.py

```diff
@@ -591,6 +638,8 @@ def verify_rif_tx_err_count(duthost, rif_interfaces, expect_counts):
     :param rif_interfaces: List of rif interface
     :param expect_counts: expected TX ERR for for every rif interface
     """
+    # Wait for the counters polling, the default interval is 1s
+    time.sleep(2)
```

Comment:
wouldn't it be more correct to query the configured interval and sleep based  on this value?

Reply:
Hi @roy-sror, added the query, please check.